src/main/scala/mesosphere/marathon/core/task/tracker/TaskTracker.scala

```diff
@@ -50,21 +53,25 @@ object TaskTracker {
 
     def hasAppTasks(appId: PathId): Boolean = appTasksMap.contains(appId)
 
-    def appTasks(appId: PathId): Iterable[MarathonTask] = {
+    def appTasks(appId: PathId): Iterable[Task] = {
       appTasksMap.get(appId).map(_.tasks).getOrElse(Iterable.empty)
     }
 
-    def marathonTask(appId: PathId, taskId: String): Option[MarathonTask] = for {
-      app <- appTasksMap.get(appId)
+    def marathonAppTasks(appId: PathId): Iterable[MarathonTask] = {
```

Comment:
Just to get the idea: we want to delete this method at some point in the future?